### ISSUE TITLE
aom: Unrecognized toolchain '-win32-gcc'

### ISSUE THREAD

**Issue by LigH-de** [jb-alvarado/media-autobuild_suite#544]
```
┌ aom git .............................................. [Updates found]
├ Running uninstall...
├ Running configure...
Likely error:
  enabling alt_intra
  enabling palette_throughput
  enabling tempmv_signaling
Unrecognized toolchain '-win32-gcc'

Configuration failed. This could reflect a misconfiguration of your
toolchains, improper options selected, or another problem. If you
don't see any useful error messages above, the next step is to look
at the configure error log file (config.log) to determine what
configure was trying to do when it died.
```

Either their error ... or maybe renamed parameter?!

[logs.zip](https://0x0.st/IKx.zip)

**Comment by wiiaboo** [comment]
No, I forgot to add the same $arch to aom as I added for vpx.

**Comment by wiiaboo** [comment]
Should  be fixed with 6825308.
